datafusion/core/tests/sql/create_drop.rs

```diff
@@ -73,3 +73,40 @@ async fn create_external_table_with_ddl() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
```

Comment:
I recommend putting these tests in https://github.com/apache/arrow-datafusion/blob/main/datafusion/core/tests/sqllogictests/test_files/ddl.slt rather than a rs test